pep-9999.rst

```diff
@@ -0,0 +1,520 @@
+  Python stable ABI and hence produced wheel archives that would still be usable
+  on the rolling beta releases.
+
+
+
+Proposal
+========
+
+The majority of the proposed changes in this PEP only affect the handling of
+pre-release versions. The one change affecting full release versions is a
+suggested change to their cadence.
+
+Changes to beta release policy, phase duration, and cadence
+-----------------------------------------------------------
+
+This PEP proposes that the policy for beta releases be adjusted to be a
+combination of the policies for alpha and beta releases:
+
+* as with current beta releases, the stable BuildBot fleet is expected to be
+  green prior to preparation and publication of the beta release
+* as with current beta releases, the release manager is expected to review
+  open release blocker issues prior to preparation and publication of the beta
+  release
+* as with current alpha releases, beta releases would *not* be considered
+  feature complete for the next X.Y.0 release
+* as with current alpha releases, all APIs added since the last CPython feature
+  release would be considered provisional
+* as with current alpha releases, beta releases would be prepared and published
+  from the master development branch
+
+
+Rather than being released monthly for a period of a few months while preparing
+a new X.Y.0 release, beta releases would instead be consistently published every
+other month.
+
+The only time this would not be the case is during the two month release
+candidate period for an upcoming X.Y.0 release.
+
+
+Changes to release candidate policy, phase duration, and cadence
+----------------------------------------------------------------
+
+Given the proposed changes to the beta release phase, the release candidate
+phase would see the following related adjustments:
+
+* Feature freeze, ABI freeze, pyc file format freeze, and maintenance branch
+  creation would all correspond with the creation of X.Y.0rc1 (currently these
+  occur across a mixture of X.Y.0b1, the last beta release, and X.Y.0rc1)
+* The X.Y.0 release candidate period would be extended from 3 weeks to 2 months
+* There would normally be two release candidates issued a month apart, but
+  additional candidates may be published at the release manager's discretion
+* The final release would occur between 1 and 4 weeks after the final release
+  candidate (depending if additional release candidates were needed after the
+  second)
+
+
+Removal of routine alpha releases
+---------------------------------
+
+Due to the proposed policy changes for beta releases and release candidates,
+CPython alpha releases would no longer serve a clear purpose, and as such
+would typically be skipped.
+
+If they chose to, release managers would still be free to make one or more
+alpha releases before entering the rolling beta release period. For example,
+that would be one way of testing out new release management tooling.
```

Comment:
See my suggestion above about the RM declaring a release as alpha rather than beta (in exceptional circumstances)

Reply:
Amended accordingly.